bridge/checkpoint/checkpoint.go

```diff
@@ -117,9 +122,12 @@ func (c *checkpoint) StartNewCheckpoint(epochSize uint64) error {
 	return nil
 }
 
-func (c *checkpoint) determineCheckpointRange(lastChildBlock, epochSize uint64) (uint64, uint64) {
-	// TODO: implement
-	return lastChildBlock + 1, lastChildBlock + epochSize
+func (c *checkpoint) determineCheckpointRange(lastChildBlock uint64) (uint64, uint64) {
+	header := c.blockchain.Header()
+
+	lastBlockNumber := header.Number
```

Comment:
You can inline this

Reply:
Thank you for suggestion, fixed b65283c9249dc562ea3597d3db2fdf69c78213fd